modules/base/src/processors/surfaceextractionprocessor.cpp

```diff
@@ -135,6 +137,7 @@ void SurfaceExtraction::process() {
             result_[i].set(method, iso, color, invert, enclose, 0.0f,
                            dispatchPool([this, vol, method, iso, color, invert, enclose,
                                          i]() -> std::shared_ptr<Mesh> {
+                            RenderContext::getPtr()->activateLocalRenderContext();
```

Comment:
lets do this for now, but I think you might be right that it would be better to add this to VolumeGL2RAMConverter or maybe texture... Not sure if this is cheep as long as there is no  actually change of context... or of even that could be expensive?